app/shared-components/PromoModal/PromoModal.jsx

```diff
@@ -76,7 +76,7 @@ class PromoModal extends React.Component {
 				break;
 			case INSIGHTS:
 				sendMessage('ping', 'promo_modals_insights_upgrade_cta');
-				url = `https://checkout.${DOMAIN}.com/insights?utm_source=gbe&utm_campaign=${utm_campaign}`;
+				url = `https://www.ghostery.com/insights/?utm_source=gbe&utm_campaign=in_app_upgrade=${utm_campaign}`;
```

Comment:
utm_campaign is a [query string](https://en.wikipedia.org/wiki/Query_string) so it is a string=string pair, so given that this function is always called with utm_campaign input as `in_app_upgrade` the current value here of utm_campaign would be `in_app_upgrade=in_app_upgrade`, i think what you want to do here is just set the value of the `utm_campaign` query param to the passed `utm_campaign` value

Reply:
^ what Frank said